ui/shared/components/buttons/EditProjectButton.jsx

```diff
@@ -4,17 +4,27 @@ import PropTypes from 'prop-types'
 
 import { updateProject } from 'redux/rootReducer'
 import UpdateButton from './UpdateButton'
-import { EDITABLE_PROJECT_FIELDS, MATCHMAKER_CONTACT_NAME_FIELD, MATCHMAKER_CONTACT_URL_FIELD } from '../../utils/constants'
+import {
+  EDITABLE_PROJECT_FIELDS,
+  PM_EDITABLE_PROJECT_FIELDS,
+  MATCHMAKER_CONTACT_NAME_FIELD,
+  MATCHMAKER_CONTACT_URL_FIELD,
+} from '../../utils/constants'
 
 const MATCHMAKER_PROJECT_FIELDS = [
-  ...EDITABLE_PROJECT_FIELDS,
-  ...[
-    { ...MATCHMAKER_CONTACT_NAME_FIELD, name: 'mmePrimaryDataOwner' },
-    { ...MATCHMAKER_CONTACT_URL_FIELD, name: 'mmeContactUrl' },
-  ].map(({ label, ...field }) => ({ ...field, label: `Matchmaker ${label}` })),
-]
+  { ...MATCHMAKER_CONTACT_NAME_FIELD, name: 'mmePrimaryDataOwner' },
+  { ...MATCHMAKER_CONTACT_URL_FIELD, name: 'mmeContactUrl' },
+].map(({ label, ...field }) => ({ ...field, label: `Matchmaker ${label}` }))
 
-const EDITABLE_FIELD_KEYS = ['projectGuid', ...MATCHMAKER_PROJECT_FIELDS.map(({ name }) => name)]
+const FIELD_LOOKUP = {
+  true: {
+    true: [...PM_EDITABLE_PROJECT_FIELDS, ...MATCHMAKER_PROJECT_FIELDS],
+    false: [...EDITABLE_PROJECT_FIELDS, ...MATCHMAKER_PROJECT_FIELDS],
+  },
+  false: { true: PM_EDITABLE_PROJECT_FIELDS, false: EDITABLE_PROJECT_FIELDS },
```

Comment:
I was confused by the `true` and `false` keys at the beginning. It could be helpful by adding comments for an example usage such as `FIELD_LOOKUP[isMmeEnabled][isPm]`.